sale_unreconciled/models/company.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    sale_reconcile_account_id = fields.Many2one(
+        "account.account",
+        domain=lambda self: [("reconcile", "=", True), ("deprecated", "=", False)],
+        string="Write-Off Account On Sales",
+        ondelete="restrict",
+        copy=False,
+        help="Write-off account to reconcile Unreconciled Sale Orders",
+    )
+
+    sale_reconcile_journal_id = fields.Many2one("account.journal")
```

Comment:
A string should be added in the definition, since the field is visible for the configuration

Reply:
If I don't add a string, it shows as "Sale Reconcile Journal". It think it is correct but I don't mind to put another string if you suggest a different thing.